person.rb

```diff
@@ -10,39 +10,37 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
```

Comment:
- [ ] Like the previous reviewer mentioned, `@rental` should be `@rentals`. kindly update this. Do not forget to update it in the setter/getter.